packages/core/src/browser/shell/tab-bar-toolbar/tab-bar-toolbar.tsx

```diff
@@ -285,6 +287,57 @@ export class TabBarToolbar extends ReactWidget {
         });
     }
 
+    /**
+     * Renders a toolbar item that is a menu, presenting it as a button with a little
+     * chevron decoration that pops up a floating menu when clicked.
+     *
+     * @param item a toolbar item that is a menu item
+     * @returns the rendered toolbar item
+     */
+    protected renderMenuItem(item: TabBarToolbarItem & MenuToolbarItem): React.ReactNode {
+        const icon = typeof item.icon === 'function' ? item.icon() : item.icon ?? 'ellipsis';
+        return <div key={item.id} className={TabBarToolbar.Styles.TAB_BAR_TOOLBAR_ITEM + ' enabled menu'}>
+            <div id={item.id} className={codicon(icon, true)} onClick={this.showPopupMenu.bind(this, item.menuPath)}
```

Comment:
Could you please add the `onClick` listener to the parent? It feels a bit odd that you cannot open the menu by clicking the chevron.

Reply:
The bigger target area for clicking is important for accessibility in any case!